static/clock.html

```diff
@@ -51,6 +52,9 @@ <h1 class="bgnow"></h1>
       var rec = xhr[0];
       var last = new Date(rec.date);
       var now = new Date( );
+      if (window.serverSettings.units == 'mmol') {
+        rec.sgv = (Math.round((rec.sgv / 18) * 10) / 10).toFixed(1);
```

Comment:
Agreed.... this phrase is now copy-pasted 4 or 5 times throughout our code base... this is a recipe for badness.